### DISCUSSION TITLE
How to install PHPMailer for multiple domains.

### DISCUSSION THREAD

**Discussion by WhiteTiger-IT** [PHPMailer/PHPMailer#2260] [Q&A]
On my server I have Apache and several VirtualHosts.
How and where should PHPMailer be installed for everyone to use?
In which folder should I run composer?

**Comment by Synchro** [comment]
I recommend that every separate site should have its own copy, and it will be updated along with all the other dependencies that each site may have. You typically run composer in the project's root folder, which will often not be the same as the public web root. The files are small, and the complexity required to share them across sites is not worth any savings you might make in disk space.
